Library/TestHelpers/TestCase.swift

```diff
@@ -54,7 +55,11 @@ internal class TestCase: FBSnapshotTestCase {
       debounceInterval: .seconds(0),
       device: MockDevice(),
       isVoiceOverRunning: { false },
-      koala: Koala(client: self.trackingClient, loggedInUser: nil),
+      koala: Koala(
+        dataLakeClient: self.dataLakeTrackingClient,
```

Comment:
There is a mock client set for this by default in `TestCase` but I think we can add tests once we whitelist events.